_posts/2021-01-13-hapi-fhir.md

```diff
@@ -62,42 +62,47 @@ Unlike the Plain Server, the JPA server provides its own database schema and han
 ### JAX-RS Server
 
 The HAPI FHIR Plain Server (RestfulServer) is implemented as a standard JEE Servlet, meaning that it can be deployed in any compliant JEE web container.
+The JAX-RS module is a community-supported module that was not developed by the core HAPI FHIR team. 
+Before deciding to use the HAPI FHIR JAX-RS module, please be aware that it does not have as complete of support for the full FHIR REST specification as the Plain Server.
 
-For users in an environment where existing services using JAX-RS have been created, 
+For users in an environment where existing JAX-RS services have been created, 
 it is often desirable to use JAX-RS for FHIR servers as well. HAPI FHIR provides a JAX-RS FHIR server implementation for this purpose.
 
 
 ## Custom Plain server implementation
 
-As you have read above, there are different server types for different needs. 
-Since the JPA implementation has everything working out of the box. 
-I thought it would be a nice challenge to convert a plain server with a NOSQL database since the JPA uses a SQL database.
+As you have read above, there are different server types for different needs.
+Since the JPA implementation is a fully out-of-the-box working implementation with a SQL database, I thought it would be a nice challenge to set up my own Plain server implementation using a NoSQL database.
+The remainder of this blogpost will cover some practical code snippets of how I've set up a FHIR Restful server using Spring Boot. 
 
-Below is a diagram of the created server compared to the JPA server:
 
-<img class="image fit" alt="custom diagram" src="/img/2021-01-23-hapi-fhir/diagram.svg">
+Below a comparison between our custom developed Plain server and an out-of-the-box JPA server:
+
+<img  class="image" style="margin:0px auto; max-width:80%" alt="custom diagram" src="/img/2021-01-23-hapi-fhir/diagram.png">
 
  
   
-JPA server:
-<img class="image fit" alt="jpa diagram" src="/img/2021-01-23-hapi-fhir/JPAdiagram.svg">
+Out-of-the-box JPA server:
+
+<img  class="image" style="margin:0px auto; max-width:80%" alt="jpa diagram" src="/img/2021-01-23-hapi-fhir/JPAdiagram.png">
  
-Compared to the JPA server that is provided out of the box, 
+Compared to the JPA server that is provided out-of-the-box, 
 there are some minor differences but by and large they are similar. 
 The differences are the database technologies and mapping technologies.
 
   
 ### Modules
 
-First of we created a simple spring boot project. Now we will load up the HAPI library. 
+First of we will create a simple Spring Boot project and load up the HAPI library using Maven.
```

Comment:
1) "First of we ..." --> "First of all, we will ..."
2) " ... and load up the ...." --> "... and load the ... "